src/core/Header/Header.tsx

```diff
@@ -16,74 +13,48 @@ import { useTheme } from '../utils/hooks/useTheme';
 import { DropdownMenu } from '../DropdownMenu';
 
 export type HeaderTranslations = {
-  notifications: string;
-  help: string;
-  myAccount: string;
   moreOptions: string;
 };
 
 export type HeaderProps = {
   /**
-   * Icon to the application.
+   * Application title.
+   * (expects `HeaderApplicationButton`)
    */
-  appIcon: React.ReactNode;
+  appButton: React.ReactNode;
   /**
-   * Name of the application.
+   * Content on the right of the application button
+   * (expects `HeaderBreadcrumbs`)
    */
-  appName?: React.ReactNode;
+  breadcrumbs?: React.ReactNode;
   /**
-   * Handler on application click
+   * Content displayed to the left of the user icon
+   * (expects array of `IconButton`, potentially wrapped in a `DropdownMenu`)
    */
-  onApplication?: () => void;
+  actions?: React.ReactNode[];
   /**
-   * `HeaderButton` array, chevron will be added between each.
-   */
-  breadcrumbsButtons?: React.ReactNode[];
-  /**
-   * Content to display in the center section of the header, should handle iui-slim class if used.
-   */
-  centerContent?: React.ReactNode;
-  /**
-   * Expects `UserIcon`, will be given a drop down if `userMenuItems` is also provided;
+   * User icon
+   * (expects a `UserIcon`, potentially wrapped in a `DropdownMenu`)
    */
   userIcon?: React.ReactNode;
-  /**
-   * Handler on notification click, will show notification bell if provided.
-   */
-  onNotification?: () => void;
-  /**
-   * Items in the help dropdown menu.
-   * Pass a function that takes the `close` argument (to close the menu),
-   * and returns a list of `MenuItem` components.
-   */
-  helpMenuItems?: (close: () => void) => JSX.Element[];
-  /**
-   * Items in the user dropdown menu.
-   * Pass a function that takes the `close` argument (to close the menu),
-   * and returns a list of `MenuItem` components.
-   */
-  userMenuItems?: (close: () => void) => JSX.Element[];
   /**
    * Items in the more dropdown menu.
    * Pass a function that takes the `close` argument (to close the menu),
    * and returns a list of `MenuItem` components.
    */
-  moreMenuItems?: (close: () => void) => JSX.Element[];
+  menuItems?: (close: () => void) => JSX.Element[];
   /**
-   * Reduce height of the header;
+   * If true, the header height is reduced, typically used when viewing 3D content.
    * @default false
    */
-  slim?: boolean;
+  isSlim?: boolean;
   /**
    * Provide localized strings.
    */
-  translatedHeader?: HeaderTranslations;
-};
+  translatedStrings?: HeaderTranslations;
+} & React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>;
 
 const headerTranslations: HeaderTranslations = {
```

Comment:
Had that because it was initialy in the HTML example, I'm not familiar with what aria-label really do. I can remove it if it is not something that is ususally there.

Reply:
After some research I'd say leave it in, but name can still be improved (i.e. "defaultTranslations").

And in the component, you can change "strings" to "headerTranslations" or anything more descriptive

Reply:
![image](https://user-images.githubusercontent.com/9084735/118166211-2ee33880-b3f3-11eb-97ba-847744091039.png)

I was talking about line 84 not line 80 😄

Reply:
I think I did what you meant, let me know if its right.